ersilia/hub/content/card.py

```diff
@@ -45,7 +46,7 @@
 except:
     Hdf5Explorer = None
 
-from ...default import CARD_FILE, METADATA_JSON_FILE, SERVICE_CLASS_FILE
+from ...default import EOS, LOCAL_CARD_FILE, METADATA_JSON_FILE, SERVICE_CLASS_FILE
```

Comment:
I don't understand where 'LOCAL_CARD_FILE' is being written to for your code to be able to read it. If it is the same information file as i mentioned above, in that case, you can reuse that same identifier instead of creating a new one.